lib/network/modules/components/Node.js

```diff
@@ -219,12 +219,10 @@ class Node {
 
     var groupObj = groupList.get(group);
 
-    // Skip merging of group font options into parent; these are required to be distinct for labels
-    // Also skip mergin of color IF it is already defined in the node itself. This is to avoid the color of the
-    // group overriding the color set at the node level
-    // TODO: It might not be a good idea either to merge the rest of the options, investigate this.
-    var skipProperties = ['font'];
-    if (newOptions !== undefined && newOptions.color !== undefined && newOptions.color != null) skipProperties.push('color');
+    // Skip any new option to avoid them being overridden by the group options.
+    var skipProperties = Object.getOwnPropertyNames(newOptions).filter(p => newOptions[p] !== undefined && newOptions[p] != null);
```

Comment:
If the first condition is true, the second will be true too. In other words the first one is useless.

Also could you use const, please? I'd like to purge var from the codebase eventually.

Reply:
Sure, thanks for the quick review.